test_cases/solidity/insufficient_gas_griefing/relayer_fixed/relayer_fixed.sol

```diff
@@ -0,0 +1,28 @@
+/*
+ * @source: https://consensys.github.io/smart-contract-best-practices/known_attacks/#insufficient-gas-griefing
+ * @author: ConsenSys Diligence
+ * Modified by Kaden Zipfel
+ */
+
+pragma solidity ^0.4.21;
+
+contract Relayer {
+    mapping (bytes => bool) executed;
+
+    function relay(bytes _data) public {
```

Comment:
Adding a `_gasLimit` argument here, and handling it in the call would probably be good to make the example clearer.